infraestructure/config/config_test.go

```diff
@@ -0,0 +1,20 @@
+package config
+
+import (
+	"testing"
+)
+
+var exp = "/Users/luis.tejeda/Documents/Source/GitHub/ondemand-go-bootcamp/poke.csv"
```

Comment:
even for testing, we shouldn't rely on absolute paths as they make the code un portable (we are never sure the machine that'll run the code will have the same folder structure), please take a look at the [Abs()](https://pkg.go.dev/path/filepath#Abs) function to get the absolute path from a relative one